tests/Fixer/PhpUnit/PhpUnitTestCaseStaticMethodCallsFixerTest.php

```diff
@@ -433,6 +433,19 @@ public function baz()
                     'call_type' => PhpUnitTestCaseStaticMethodCallsFixer::CALL_TYPE_THIS,
                 ],
             ],
+            [
+                <<<'EOF'
+<?php
+class HavingPropertyWithNameAsMethodToUpdateTest extends PHPUnit
+{
+    public function foo()
+    {
+        $this->attribute = 42;
```

Comment:
Test and fix is ok, but I think it isn't 100% clear enough what you are testing, for someone who look at this the first time. The class name of course helps, but I would use a more wide spread `$this->assertSame = 42` for example.

Reply:
:+1:  we can have both I think :)